csrc/scheduler/vectorize_helper.cpp

```diff
@@ -778,7 +775,7 @@ Val* ContiguousInnerDimensionsMapper::getContigMergeOfInnerSize(
     }
 
     // Mapping order isn't correct, cannot expand vectorization dimension.
-    if (!projected_dims[--projected_dims_i]->sameAs(root_id)) {
+    if (projected_dims[--projected_dims_i] != root_id) {
```

Comment:
This change is necessary when I updated the transpose scheduler to user this.

We are accidentally mapping constantly size dimensions other wise. Which were luckily causing error during getProjectedExtent.